### ISSUE TITLE
About Training Computation Costs

### ISSUE THREAD

**Issue by jzliu02** [WU-CVGL/SIU3R#6]
"4.1 Experimental Setup
Implementation Details. We utilize ScanNet[17] for training and validation, the largest public
dataset that concurrently provides multi-view images with dense semantic/instance segmentation
labels and text-referred segmentation labels[56]. **We conduct training on 8 NVIDIA GeForce RTX
4090 GPUs, with our model trained for 100 epochs using a per-GPU batch size of 3 (total batch size
of 24) for about 2 hours**. AdamW optimizer[57] is employed with an initial learning rate of 1e-4
followed by cosine decay scheduling."

Hi, thanks for your great work! But I feel surprised about the training cost.
I am not familiar with the task. But do you mean you train the **whole model** within **several hours** using 8 RTX4090 GPUs, and you complete 100 epochs of ScanNet dataset? That is so efficient.

**Comment by insomniaaac** [comment]
in fact, i freeze image encoder from mast3r.
trainable components are vit adapter, unified decoder and gaussian decoder.
